CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# Contributing <!-- omit in toc -->
+
+First of all, thank you for contributing to MeiliSearch! The goal of this document is to provide everything you need to know in order to contribute to MeiliSearch and its different integrations.
+
+- [Assumptions](#assumptions)
+- [How to Contribute](#how-to-contribute)
+- [Development Workflow](#development-workflow)
+- [Git Guidelines](#git-guidelines)
+- [Release Process (for internal team only)](#release-process-for-internal-team-only)
```

Comment:
Where is this part? At least the bors part since you've installed it ;) And the sentence about the semantic versionning
https://github.com/meilisearch/integration-guides/blob/main/templates/CONTRIBUTING.md#release-process-for-internal-team-only

Reply:
Ah yes! Absolutely